coralproject/talk
# [CORL-460] Allow custom email message for user bans

## What does this PR do?
Adds an optional textarea in the ban confirmation modal allowing the moderator to customize the email sent to the banned user. 

## How do I test this PR?
- log in as moderator
- select "ban user" from the user status dropdown in the community tab
- click the "edit email" checkbox
- enter a custom message
- confirm ban
- check email message to find custom text. 

**Note** compare is vs #2419 because it relies on shared code, will rebase vs next when it's merged.
